pkg/controllers/disruption/consolidation_test.go

```diff
@@ -608,69 +623,112 @@ var _ = Describe("Consolidation", func() {
 		})
 	})
 	Context("Replace", func() {
-		It("can replace node", func() {
-			labels := map[string]string{
-				"app": "test",
-			}
-			// create our RS so we can link a pod to it
-			rs := test.ReplicaSet()
-			ExpectApplied(ctx, env.Client, rs)
-			Expect(env.Client.Get(ctx, client.ObjectKeyFromObject(rs), rs)).To(Succeed())
-
-			pod := test.Pod(test.PodOptions{
-				ObjectMeta: metav1.ObjectMeta{Labels: labels,
-					OwnerReferences: []metav1.OwnerReference{
-						{
-							APIVersion:         "apps/v1",
-							Kind:               "ReplicaSet",
-							Name:               rs.Name,
-							UID:                rs.UID,
-							Controller:         ptr.Bool(true),
-							BlockOwnerDeletion: ptr.Bool(true),
-						},
-					}}})
-			ExpectApplied(ctx, env.Client, rs, pod, node, nodeClaim, nodePool)
+		DescribeTable("can replace node",
+			func(isSpotNode bool) {
+				if isSpotNode {
+					node = spotNode
+					nodeClaim = spotNodeClaim
+					ctx = options.ToContext(ctx, test.Options(test.OptionsFields{FeatureGates: test.FeatureGates{SpotToSpotConsolidation: lo.ToPtr(true)}}))
```

Comment:
Is it reasonable to say that these tests should all work even if spot to spot is enabled? Can we just enable it at the top level for all tests?

Reply:
Done.